docs/faq.md

```diff
@@ -40,6 +111,12 @@ in your Dockerfile.
 * If you are able, modify the application that you're running to
 add an explicit signal handler for `SIGTERM`.
 
+* Set the `stop_signal` to a signal which the application knows how to handle:
+
+      web:
+        build: .
+        stop_signal: SIGTERM
```

Comment:
`SIGTERM` is already the default, so I think this should be `SIGKILL` or `SIGINT`?